MyProject/cls/Employee.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace cls
+{
+    class Employee: Person
+    {
+        public string Position { get; private set; }
+        public int AccessLevel { get; private set; }
+
+        private double Salary;
+
+        public int userID { get; private set; }
+
+        public void SetSalary(double Salary)
```

Comment:
you have two methods named "SetSalary" and they do different stuff, please fix.

Reply:
Done, one supposed to be Get instead of Set